.github/workflows/rc-release.yaml

```diff
@@ -0,0 +1,60 @@
+name: Release to @rc-<name> tag on npm
+
+on:
+  push:
+    branches:
+      - "rc/*"
+
+jobs:
+  release-pr:
+    name: "Release RC to npm"
+    runs-on: ubuntu-latest
+    permissions: write-all
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          ref: ${{ github.event.pull_request.head.ref }}
+
+      - name: CI Setup
+        uses: ./.github/actions/ci-setup
+
+      - name: Ensure NPM access
+        run: npm whoami
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      - name: Build
+        run: pnpm build
+
+      - name: Get RC name
+        uses: frabert/replace-string-action@v2
+        id: rc_name
+        with:
+          string: ${{ github.ref }}
+          pattern: "rc/"
+          replace-with: ""
+
+      - name: Release to @${{ steps.rc_name.outputs.replaced }} tag on npm
+        id: release
+        run: |
+          echo "${{ steps.rc_name.outputs.replaced }}"
+          pnpm changeset:next
+          git add .changeset/fuel-labs-ci.md
+          pnpm changeset version --snapshot rc-${{ env.RC_NAME }}
```

Comment:
What's up with `env.RC_NAME`?

Shouldn't it be `steps.rc_name.outputs.replaced`?

Reply:
Correct, apologies this was left in when I was testing using a hardcoded branch name for `rc-salamander` - https://www.npmjs.com/package/fuels/v/0.0.0-rc-salamander-20231214170936